python/ray/tests/conftest.py

```diff
@@ -1048,7 +1048,7 @@ def append_short_test_summary(rep):
 
     summary_dir = os.environ.get("RAY_TEST_SUMMARY_DIR")
 
-    if platform.system() != "Linux":
+    if platform.system() == "Darwin":
```

Comment:
I thought we mostly use `sys.platform` to check platform (which is `"win32"` on windows), rather than using `platform.system()`

Reply:
I have been using `platform.system()` more in my code as well; they tends to be less surprising (e.g. linux instead of linux-gnu???)

Reply:
I guess it does not really matter.

I just saw other places in this repo using `sys.platform`; not a big deal.